Example/FancyShop/Pods/Target Support Files/Pods-FancyShop/Pods-FancyShop-frameworks.sh

```diff
@@ -46,8 +46,8 @@ install_framework()
   fi
 
   # Use filter instead of exclude so missing patterns don't throw errors.
```

Comment:
You can revert this file, actually we should stop tracking anything inside Pods folder

Reply:
reverted

Reply:
don't think these are showing up in changed files now